src/google/protobuf/io/io_win32.cc

```diff
@@ -357,6 +362,92 @@ wstring testonly_utf8_to_winpath(const char* path) {
   return as_windows_path(path, &wpath) ? wpath : wstring();
 }
 
+int ExpandWildcards(
+    const string& path, std::function<void(const string&)> consume) {
+  if (path.find_first_of("*?") == string::npos) {
+    // There are no wildcards in the path, we don't need to expand it.
+    consume(path);
+    return ExpandWildcardsResult::kSuccess;
+  }
+
+#ifdef SUPPORT_LONGPATHS
```

Comment:
Isn't `SUPPORT_LONGPATHS` always defined here since it's set on line 50?

Reply:
Great, I would say you can just delete a lot of this code since we only need to handle the `SUPPORT_LONGPATHS` case.

Reply:
Done.